.github/workflows/main.yml

```diff
@@ -48,17 +49,19 @@ jobs:
           python -m tox -e lint
 
   test-pypi:
-    name: Test with Python${{ matrix.python-version }} (Python${{ matrix.python-version }} + tox)
+    name: Test with Python${{ matrix.python-version }} (tox, ${{ matrix.os }})
     needs: lint
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ 'ubuntu-latest' ]
         python-version:
           - "3.8"
           - "3.9"
           - "3.10"
           - "3.11"
           - "3.12"
+          # - "3.13"
```

Comment:
py313 is added below, should this be uncommented?

Reply:
Python3.13 was released yesterday! I'll try to see if this works (previously, this would fail).